tests/integration/suite/test_workloads.py

```diff
@@ -310,6 +310,31 @@ def test_statefulset_workload_volumemount_subpath(admin_pc):
         assert e.value.error.status == 422
 
 
+def test_workload_redeploy(admin_pc):
+    client = admin_pc.client
+    ns = admin_pc.cluster.client.create_namespace(
+        name=random_str(),
+        projectId=admin_pc.project.id)
```

Comment:
can we add `remove_resource` for this namespace and the workload on line 319 too, we won't have to use client.delete